build.xml

```diff
@@ -372,6 +372,29 @@
     <exec command="php ${srcdir}/composer.phar install ${composer_extra_params}" passthru="true" checkreturn="true" />
   </target>
 
+  <!-- Copy files from node modules to themes -->
+  <target name="copynodemodules">
```

Comment:
This Phing action should no longer exist after the latest merge with dev; it has been replaced by a different mechanism. I think it just needs to be deleted here. Any new copy operations should be added to https://github.com/vufind-org/vufind/blob/dev/themes/bootstrap3/tools/copyDependencies.mjs

Reply:
That explains it! I was confused by this diff.